Makefile

```diff
@@ -42,7 +42,8 @@
 
 ############################## TOOLCHAIN ###############################
 #
-CSC         = mcs -sdk:4.0
+SDK        :=
```

Comment:
@Mailaender: see https://www.gnu.org/software/make/manual/html_node/Setting.html#Setting. I never had an issue with overriding a make variable from the command line, even when ?= wasn't used, though.